tooling/ssa_fuzzer/src/lib.rs

```diff
@@ -55,21 +57,19 @@ mod tests {
         }
     }
 
-    /// Generates a random array with config::NUMBER_OF_VARIABLES_INITIAL elements
-    fn generate_values() -> Vec<u64> {
+    fn generate_values<T>() -> (T, T)
```

Comment:
I never knew why the number of constant `NUMBER_OF_VARIABLES_INITIAL` was used in so many places, but perhaps it would be good to include a docstring about why a generating random _pair_ of values has a dedicated method.